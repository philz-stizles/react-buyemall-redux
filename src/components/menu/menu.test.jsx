import { shallow } from 'enzyme'
import configureStore from 'redux-mock-store';
import React from 'react'
import Menu from './menu.component'

it('should render Menu with given state from Redux store', () => {
    const initialState = {
        menu: {
            items: [],
        }
    };

    const mockStore = configureStore()
    const store = mockStore(initialState)

    expect(shallow(<Menu store={store}/>)).toMatchSnapshot()
})