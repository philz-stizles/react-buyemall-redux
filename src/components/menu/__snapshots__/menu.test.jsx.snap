// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Menu with given state from Redux store 1`] = `ShallowWrapper {}`;

exports[`should render with given state from Redux store 1`] = `ShallowWrapper {}`;
